src/main/webapp/WEB-INF/web.xml

```diff
@@ -3,4 +3,9 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
+
```

Comment:
Videl som ze sem niektori uvadzali servlety ako zoznam. Neviem ale ako to ma funkciu.

Reply:
Hmmm možno pre SEO vyhľadávačov, ale môžem vyzistiť.